samples/admin/ADMINReqMoveComplete/ADMINReqMoveComplete.c

```diff
@@ -0,0 +1,500 @@
+    COLLECTIONPOSITION colPosition;                              /* collection position */
+    NOTEID      viewID = 0;                                      /* view ID */ 
+    DHANDLE     hBuffer = NULLHANDLE;                            /* handle to buffer of note ids */
+    DWORD       dwEntriesFound = 0;                              /* number of entries found */
+    NOTEID*     IdList = NULL;                                   /* pointer to a note id list */
+    WORD        wSignalFlag = 0;                                 /* signal and share warning flags */
+    DWORD       i=0;                                             /* local variable */
+    DWORD       notesFound=0;                                    /* notes found variable */
+    DBHANDLE    hNABook = NULLHANDLE;                            /* handle to NAB */
+    char       *pszNABFile = NULL;                               /* pointer to NAB file*/
+    char        szNABFilePath[MAXPATH] = {0};                    /* NAB file path*/
+    char        szUserName[] = "CN=test user/O=HCLPNP";          /* user to be renamed*/
+    char        szUserFld[] = "$Users";                          /* view to search for the user*/
+    char        szDBName[MAXPATH]= "names.nsf";                  /* NAB database name*/
+    NOTEID      noteID = 0;                                      /* noteID */
+	
+    pszServerName = szServer;
+    pszAdmin4NSFFile = szAdminDBName;
+    pszNABFile = szDBName;
+	
+
+    /* Process input arguments. */
+    ProcessArgs(argc, argv, pszServerName, pszAdmin4NSFFile);
+	
+    /* Initialize the notes. */
+    if (error = NotesInitExtended(argc, argv))
+    {
+        PRINTERROR (error,"NotesInitExtended");
+        return(1);
+    }
+	
+    /* Reading Domino data directory. */
+    OSGetDataDirectory(szDataPath);
+    PRINTLOG("data path : %s\n", szDataPath);
+
+    /* Adding trailing path separator. */
+    if (error = OSPathAddTrailingPathSeparator(szDataPath, sizeof(szDataPath)-1))
+    {
+        PRINTERROR(error,"OSPathAddTrailingPathSeparator");
+        NotesTerm();
+        return (1);
+    }
+    PRINTLOG("data path with trailing path separator: %s\n", szDataPath);
+
+    /* Preparing path for cert ID. */
+    strncat(szDataPath, szOrgUnitCertID, sizeof(szDataPath)-1 - strlen(szDataPath));
+    pszCertIDFilePath = szDataPath;
+    PRINTLOG("cert ID path : %s\n", pszCertIDFilePath);
+
+    /* Construct the path for the admin request file. */
+    if (error = OSPathNetConstruct(NULL, pszServerName, pszAdmin4NSFFile, szAdminFilePath))
+	{
+        PRINTERROR(error,"OSPathNetConstruct");
+        NotesTerm();
+        return (1);
+    }
+	
+    /* Open the admin4 DB database. */
+    if (error = NSFDbOpen(szAdminFilePath, &db_handle))
+    {	
+	PRINTERROR (error,"NSFDbOpen");
+	NotesTerm();
+	return (1);
+    }
+	
+     /* Construct the path for the NAB file */
+    if (error = OSPathNetConstruct(NULL, pszServerName, pszNABFile, szNABFilePath))
```

Comment:
If this API fails db_handle handle will be left open, need to close the handle.

Reply:
Handled now.